src/main/resources/application.properties

```diff
@@ -0,0 +1,17 @@
+
+profile.active=prod
+
+#Bucket
+amazon.clientS3.accesskey = ${S3_ACCESS_KEY}
+amazon.clientS3.secretkey = ${S3_SECRET_KEY}
+amazon.clientS3.region = ${S3_REGION}
+amazon.bucketS3.name = ${S3_BUCKET_NAME}
+amazon.bucketS3.key = ${S3_BUCKET_KEY}
+
+#SQS
+amazon.sqs.region.static= ${SQS_REGION}
+amazon.sqs.credentials.accessKey= ${SQS_ACCESS_KEY}
+amazon.sqs.credentials.secretKey= ${SQS_SECRET_KEY}
+amazon.sqs.end-point.services-queue= ${SQS_SERVICES_QUEUE}
```

Comment:
is this the queue url or something related to aws service?

Reply:
Yes, we changed the secret to be ${SQS_ENDPOINT_SERVICES_QUEUE} to be more precise about the fact that is the Endpoint of the queue